google-cloud-logging/src/main/java/com/google/cloud/logging/Logging.java

```diff
@@ -668,6 +668,197 @@ public static EntryListOption filter(String filter) {
+   * }</pre>
+   *
+   * @return the created exclusion
+   * @throws LoggingException upon failure
+   */
+  Exclusion create(Exclusion exclusion);
+
+  /**
+   * Sends a request for to create the exclusion. This method returns a {@code ApiFuture} object to
+   * consume the result. {@link ApiFuture#get()} returns the created exclusion.
+   *
+   * <p>Example of asynchronously creating the exclusion.
+   *
+   * <pre>{@code
+   * String exclusionName = "my_exclusion_name";
+   * Exclusion exclusion = Exclusion.of(exclusionName, "resource.type=gcs_bucket severity<ERROR sample(insertId, 0.99)");
+   * ApiFuture<Exclusion> future = logging.createAsync(exclusion);
+   * // ...
+   * Exclusion exclusion = future.get();
+   * }</pre>
+   */
+  ApiFuture<Exclusion> createAsync(Exclusion exclusion);
+
+  /**
+   * Gets the description of an exclusion or {@code null} if not found.
+   *
+   * <p>Example of getting the description of an exclusion.
+   *
+   * <pre>{@code
+   * String exclusionName = "my_exclusion_name";
+   * Exclusion exclusion = logging.getExclusion(exclusionName);
+   * if (exclusion == null) {
+   *   // exclusion was not found
+   * }
+   * }</pre>
+   *
+   * @throws LoggingException upon failure
+   */
+  Exclusion getExclusion(String exclusion);
+
+  /**
+   * Sends a request for to get the description of an exclusion . This method returns a {@code
+   * ApiFuture} object to consume the result. {@link ApiFuture#get()} returns the requested
+   * exclusion or {@code null} if not found.
+   *
+   * <p>Example of asynchronously getting the exclusion.
+   *
+   * <pre>{@code
+   * String exclusionName = "my_exclusion_name";
+   * ApiFuture<Exclusion> future = logging.getExclusionAsync(exclusionName);
+   * // ...
+   * Exclusion exclusion = future.get();
+   * if (exclusion == null) {
+   *   // exclusion was not found
+   * }
+   * }</pre>
+   *
+   * @throws LoggingException upon failure
+   */
+  ApiFuture<Exclusion> getExclusionAsync(String exclusion);
+
+  /**
+   * Updates one or more properties of an existing exclusion.
+   *
+   * <p>Example of updating the exclusion.
+   *
+   * <pre>{@code
+   * String exclusionName = "my_exclusion_name";
+   * Exclusion exclusion = Exclusion.newBuilder(exclusionName, "resource.type=gcs_bucket severity<ERROR sample(insertId, 0.99)")
+   *     .setDescription("new description")
+   *     .setIsDisabled(false)
+   *     .build();
+   * Exclusion exclusion = logging.update(exclusion);
+   * }</pre>
+   *
+   * @return the updated exclusion
+   * @throws LoggingException upon failure
+   */
+  Exclusion update(Exclusion exclusion);
+
+  /**
+   * Sends a request to change one or more properties of an existing exclusion. This method returns
+   * a {@code ApiFuture} object to consume the result. {@link ApiFuture#get()} returns the updated
+   * exclusion or {@code null} if not found.
+   *
+   * <p>Example of asynchronously updating the exclusion.
```

Comment:
How about `Example of asynchronous exclusion update`?

Reply:
done